prac_06/guitar.py

```diff
@@ -0,0 +1,16 @@
+class Guitar:
+    def __init__(self, name: str, year: int, cost: float):
+
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self) -> str:
+        return f"{self.name} ({self.year}), Cost: ${self.cost:.2f}"
+
+    def get_age(self) -> int:
+        current_year = 2024
+        return current_year - self.year
+
+    def is_vintage(self) -> bool:
+        return self.get_age() >= 50
```

Comment:
Same with the previous two functions, I'm unsure why with the is_vintage function, the "-> bool" is there. Other than that, this all looks well done.